cmd/notary-server/config.json

```diff
@@ -11,5 +11,8 @@
 	},
 	"logging": {
 		"level": 5
+	},
+	"storage": {
+		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
```

Comment:
This is the only config we have for mysql for now. I'm leaving it there so that when I replace it I have a reminder of the mysql connection url.

It should probably also be replicated into the dev config file.